docs/administrative-requirements.md

```diff
@@ -106,25 +101,50 @@ Resources:
 
-## ESECC
+## VASI and eMASS 
 
-A request to the Enterprise Security External Change Council (ESECC) may be needed to authorize an opening in the firewall to enable a connection between the client system and the Enterprise Event Bus.
+The client system should either have or be in the process of procuring: 
 
-Prior to determining if your system will require an ESECC, teams are highly encouraged to attempt connecting to the Enterprise Event Bus. If an error message indicates that traffic is blocked at the network level, then check to see if an ESECC is required.
+* a VASI ID, which identifies your system in VA, and
+* an eMASS ID
 
-![A diagram showing various scenarios indicating whether a system would need an ESECC. If you are already on the Lighthouse Delivery Infrastructure (LHDI), no ESECC is required. If you are on AWS GovCloud but outside of LHDI, check to see if an ESECC is required. If you are on Non-AWS Cloud (e.g. Azure), check to see if an ESECC is required. If you are on-premises, e.g., VistA, an ESECC is required.](img/Client-Environments-ESECC-Decision-Circles.svg)
+**Note**: The VASI and eMASS Processes have merged and are now accomplished via a single LEAF intake (formerly GRC intake) request: [Unified System Registry Intake (must be on VA network to view)](https://leaf.va.gov/NATIONAL/103/cybersecurity_request_portal/).
 
-Resources:
+### VASI
 
-* [ESECC Public URL process (must be part of VA GitHub organization to view)](https://github.com/department-of-veterans-affairs/devops/blob/master/docs/ESECC-Public-URL-process.md#prereq)
-* [Tutorial video (must be on VA network to view)](https://dvagov.sharepoint.com/:v:/r/sites/OITECOESDKM/KM%20video%20library/Stream%20Migrated%20Videos/Knowledge%20Management%20Videos/KB0106755%20-%20Network%20ESECC%20-%20Create%20a%20Change%20Request-20221206_034448.mp4?csf=1&web=1&e=cCAS5z&nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D) on creating a Change Request in ESECC
-* [ESECC Self-service Portal (must be on VA network to view)](https://esecc.va.gov/CGWeb/Main.aspx) for initiating new requests or change requests
+The VA Systems Inventory (VASI) is intended to be a registry of all applications in use at VA. 
+
+The Enterprise Event Bus VASI ID is 3325, and [this is our entry (must be on VA network to view)](https://vaww.vear.ea.oit.va.gov/system_and_application_domain_defs_system_381405.htm) in the VASI registry. You may find your system’s VASI ID by visiting the [VA System Inventory (must be on VA network to view)](https://vaww.vear.ea.oit.va.gov/).
+
+**For non-LHDI clients**:
+
+* If a prospective client system does not have a VASI ID, it will need to submit a LEAF intake request to get one. 
+
+**For new LHDI clients**:
```

Comment:
Same conceptual comment as above re what if the client is new to Event Bus and is an existing LHDI tenant.

Reply:
Addressed this in a previous comment. Let me know if you agree.

Reply:
Yes, as noted above in the previous comment. Add a header and wordsmith as suggested in my previous comment to remove the potential ambiguity around the current wording of "LHDI clients."

Reply:
This has been updated for all sections, including some changes to formatting and clean-up.